pkg/daemon/ceph/client/mon.go

```diff
@@ -66,3 +74,60 @@ func GetMonQuorumStatus(context *clusterd.Context, clusterInfo *ClusterInfo) (Mo
 
 	return resp, nil
 }
+
+// EnableStretchElectionStrategy enables the mon connectivity algorithm for stretch clusters
+func EnableStretchElectionStrategy(context *clusterd.Context, clusterInfo *ClusterInfo) error {
+	args := []string{"mon", "set", "election_strategy", "connectivity"}
```

Comment:
What about making `"connectivity"` as an argument so we make the function more flexible?